bigtable-hbase-mirroring-client-1.x-parent/bigtable-hbase-mirroring-client-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/mirroring/utils/Helpers.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase.mirroring.utils;
+
+import com.google.common.primitives.Longs;
+import java.io.IOException;
+import org.apache.hadoop.hbase.client.Append;
+import org.apache.hadoop.hbase.client.Delete;
+import org.apache.hadoop.hbase.client.Get;
+import org.apache.hadoop.hbase.client.Increment;
+import org.apache.hadoop.hbase.client.Mutation;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.client.RowMutations;
+import org.apache.hadoop.hbase.client.Scan;
+
+public class Helpers {
+
+  public static Put createPut(byte[] row, byte[] family, byte[] qualifier, byte[] value) {
+    Put put = new Put(row);
+    put.addColumn(family, qualifier, value);
```

Comment:
This is not true, we are not adding timestamps ourselves.

Reply:
I see, we should add into the docs that USERS SHOULD ADD TIMESTAMPS.

Reply:
Ideally, it should be configurable, where in they can add a config and mirrroring client will add timestamps so that consistency is maintained.

Reply:
I'll implement automatic timestamping for `Put`s when a flag is set.

I see that HBase 1.x doesn't support setting a timestamp for `Increment`s and `Append`s, this seems to be not supported by Bigtable either, thus I'll leave them without modification.

I'm not sure if I should set timestamps in deletes - as per `DeleteAdapter.java` support of deletes with versions is limited in Bigtable, only point and column deletes are allowed.
Point deletes should have a version specified by the user, so we cannot modify it without changing the expected behavior.
Thus column deletes are the only case in which we can add a timestamp when it is missing and ensure consistent modification to both databases, but I cannot convince myself that adding the timestamp in that single case is worth it.

I'll use a monotonic clock as a source for timestamps (and, as far as I can see, there is no standard monotonic clock in java, thus I'll use a snapshot of `System.getCurrentTimeMillis()` and a `Stopwatch` to measure millis elapsed since the snapshot).

Reply:
Timestamps should only be added for PUTs. 

Timestamps in deletes are tricky and have a different meaning. 

I am assuming we fail fast for deletes that are not supported by Bigtable (https://cloud.google.com/bigtable/docs/hbase-differences#mutations_and_deletions)? or they are written to the failed mutation log?